packages/frontend/src/components/UserSettings/DeleteOrganisationPanel/index.tsx

```diff
@@ -0,0 +1,105 @@
+import { Button, InputGroup, Intent } from '@blueprintjs/core';
+import { MenuItem2, Popover2 } from '@blueprintjs/popover2';
+import { Organisation } from '@lightdash/common';
+import React, { FC, useEffect, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { useOrganisation } from '../../../hooks/organisation/useOrganisation';
+import { useDeleteOrganisationMutation } from '../../../hooks/organisation/useOrganisationDeleteMultation';
+import { useApp } from '../../../providers/AppProvider';
+import { ErrorMessage } from '../../common/modal/ActionModal';
+import BaseModal from '../../common/modal/BaseModal';
+import {
+    CardContainer,
+    Description,
+    PanelContent,
+    Title,
+} from './DeleteOrganisationPanel.styles';
+
+export const DeleteOrganisationPanel: FC = () => {
+    const { isLoading: isOrgLoading, data: organisation } = useOrganisation();
+    const { mutate, isLoading: isDeleting } = useDeleteOrganisationMutation();
+
+    const [showConfirmation, setShowConfirmation] = useState<boolean>(false);
+    const [confirmOrgName, setConfirmOrgName] = useState<string>();
+    const { user } = useApp();
+
+    if (isOrgLoading || organisation === undefined) return null;
+    if (!user.data?.ability?.can('delete', 'Organization')) return null;
+
+    return (
+        <>
+            <CardContainer>
+                <div>
+                    <Title>Danger zone </Title>
+                    <Description>
+                        This action deletes the whole workspace and all its
+                        content, including users. This action is not reversible.
+                    </Description>
+                </div>
+                <PanelContent>
+                    <Button
+                        outlined
+                        intent={Intent.DANGER}
+                        icon="delete"
```

Comment:
maybe a 🗑️ icon?

Reply:
The original design was using `delete` I don't mind which one we use... @PriPatel  ? 

![image](https://user-images.githubusercontent.com/1983672/212870126-bd1a4d9c-e10a-49e2-aa82-ce59f9698cb0.png)

Reply:
Ah that icon is what we use across the Project Settings to delete users and stuff (even though it sucks!). If its ok to change the particular one to the trash icon, that would be better! TY @IrakliJani and @rephus ! <3

Reply:
Done 
![image](https://user-images.githubusercontent.com/1983672/212926507-cf4b26e7-3a94-4237-ab1c-c5b53ac0b808.png)

Reply:
Done 
![image](https://user-images.githubusercontent.com/1983672/212926594-e3a81322-d4da-42a6-a51d-440736a57a5e.png)